src/components/EventList/components/EventItem/index.jsx

```diff
@@ -32,6 +36,18 @@ const EventItem = ({ event }) => {
           <Typography type="caption" className={styles.descriptionText}>
             {event.Description || 'No description available'}
           </Typography>
+          <Button onClick={() => alert(event.date +" " +event.timeRange)}
+          >name</Button>
+        <AddToCalendarButton
+          name= {event.title}
+          options={['Apple','Google']}
+          location= {event.location}
+          startDate="2023-06-04"
+          endDate="2023-06-04"
+          startTime="10:15"
+          endTime="23:30"
+          timeZone="America/Indiana/Indianapolis"
```

Comment:
This is not our timezone. That would be an hour behind us.

I believe you would want "America/New York" since all of our events thus far are in that timezone.